modules/analytics/src/views/full/index.tsx

```diff
@@ -129,12 +139,15 @@ const Analytics: FC<any> = ({ bp }) => {
       const channels = data
         .map(x => x.name)
         .filter(x => x.startsWith('channel'))
-        .map(x => x.replace('channel-', ''))
+        .map(x => {
+          const channel = x.replace('channel-', '')
+          return { value: channel, label: capitalize(channel) }
+        })
 
       setChannels(prevState => [...prevState, ...channels])
     })
 
-    dispatch({ type: 'datesSuccess', data: { dateRange: [thisWeek, now] } })
+    dispatch({ type: 'datesSuccess', data: { dateRange: [last7days, now] } })
```

Comment:
why remove this week ?

Reply:
In terms of usage, I prefer to see current week more than last 7 days.

Reply:
This was asked by Nadine, so I guess that's what clients want to see